net/src/Sails.Remoting.Abstractions/IRemoting.cs

```diff
@@ -2,28 +2,63 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Substrate.Gear.Api.Generated.Model.gprimitives;
+using Substrate.NetApi.Model.Types;
 using GasUnit = Substrate.NetApi.Model.Types.Primitive.U64;
 using ValueUnit = Substrate.NetApi.Model.Types.Primitive.U128;
 
 namespace Sails.Remoting.Abstractions;
 
 public interface IRemoting
 {
-    Task<(ActorId ProgramId, byte[] EncodedReply)> ActivateAsync(
+    /// <summary>
+    /// Sets account for signing transactions.
+    /// </summary>
+    /// <param name="signingAccount"></param>
+    void SetSigningAccount(Account signingAccount);
```

Comment:
I've decided to put it here in order to avoid any re-connections. Might not needed here at all. The actual design depends on the usage - I would expect it to be used per-scope where one don't need to change user at all.